sycl/doc/extensions/proposed/sycl_ext_oneapi_composite_device.asciidoc

```diff
@@ -218,6 +218,18 @@ This definition means that a SYCL context that contains a composite device is
 compatible with any of its component devices, even if those component devices
 are not contained by the context.  See the core SYCL specification for details.
 
+=== Interaction with the default context
+
+If the implementation supports the
+link:../supported/sycl_ext_oneapi_default_context.asciidoc[
+sycl_ext_oneapi_default_context] extension, then each platform has the notion
+of a default context.  This default context contains only the platform's root
+devices, which are those returned by `platform::get_devices`.  Any composite
+devices for the platform are not considered root devices, and thus the default
+context does not include these composite devices.  If an application wants a
+context that contains the composite devices, then it must explicitly create
+such a context.
```

Comment:
> I think we were leaning towards this semantic, and it also seems like the safe choice for now. If we decide later that the default context _should_ contain the component devices, then we can change the spec later without breaking any existing code.

You mean composite rather than component in your comment, right?

Reply:
Sorry, I think I missed your response.  Yes, I had a typo in my comment -- I meant "composite" instead of "component".  Here is my comment again with the typo fixed:

> I think we were leaning towards this semantic, and it also seems like the safe choice for now. If we decide later that the default context _should_ contain the composite devices, then we can change the spec later without breaking any existing code.

Note that the changes in the PR are OK.  It was only my PR comment that was wrong.